framework/pym/play/commands/autotest.py

```diff
@@ -148,12 +148,20 @@ def autotest(app, args):
         opener.open('%s://localhost:%s/@kill' % (protocol, http_port))
     except Exception as e:
         pass
- 
+
+    testCompleted = False
```

Comment:
In my opinion introducing the new variable is not necessary. The `testspassed` (line 157) should be used instead.

Reply:
Agreed, it can be solved with one variable.  Agreed., it is a matter of opinion whether it should be included. My reasoning for providing it was to speak to three potential states of interest: 1) tests failed to run; 2) tests ran but failed; and 3) tests ran and passed. A single boolean loses this level of resolution.   Whatever you decide to do here is fine with me.